app/controllers/hiring_staff/vacancies/documents_controller.rb

```diff
@@ -1,5 +1,14 @@
 class HiringStaff::Vacancies::DocumentsController < HiringStaff::Vacancies::ApplicationController
-  def index
-    # WIP
+  skip_before_action :verify_authenticity_token
+  before_action :school, :redirect_unless_vacancy_session_id, only: %i[new create]
+
+  def new
```

Comment:
I would keep this as `index` more than `new`, the reason is that it is actually an index of documents with a form to upload/edit documents.